package.json

```diff
@@ -59,7 +59,9 @@
     "vitest": "latest"
   },
   "dependencies": {
+    "@ai-sdk/anthropic": "^0.0.42",
```

Comment:
It seems that we can't currently pass `anthropic` from `ai-sdk` as a model to `RAGChat` because of a type mismatch. Let's either fix this (we can do something similar for `AnthropicMessagesLanguageModel` as we did with `OpenAIChatLanguageModel`) and add a test for it, or remove this dependency and open another issue for `@ai-sdk/anthropic` support.